presto-hive-common/src/main/java/com/facebook/presto/hive/util/DirectoryLister.java

```diff
@@ -11,18 +11,25 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.facebook.presto.hive;
+package com.facebook.presto.hive.util;
```

Comment:
`DirectoryLister` should belong to `filesystem`

Reply:
Will remove the util package completely.